File: server/src/main/java/com/kulygin/dto/list/NewListDto.java
package com.kulygin.dto.list;

import com.kulygin.dto.NewDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class NewListDto {

    private List<NewDto> news;
    
}

File: client/src/main/web/src/app/dto/list/new-list-dto.ts
import {NewDto} from '../new-dto';

export class NewsListDto {

    private _news: NewDto[] = [];

    constructor(news: NewDto[]) {
        this._news = news;
    }

    get news(): NewDto[] {
        return this._news;
    }

    set news(value: NewDto[]) {
        this._news = value;
    }

}
